pkg/crc/preflight/preflight_linux.go

```diff
@@ -114,10 +87,30 @@ var libvirtPreflightChecks = [...]Check{
 }
 
 func getPreflightChecks(experimentalFeatures bool) []Check {
-	checks := []Check{}
+	return getPreflightChecksForDistro(distro(), experimentalFeatures)
+}
+
+func getPreflightChecksForDistro(distro crcos.OsType, experimentalFeatures bool) []Check {
+	var checks []Check
 	checks = append(checks, genericPreflightChecks[:]...)
 	checks = append(checks, nonWinPreflightChecks[:]...)
 	checks = append(checks, libvirtPreflightChecks[:]...)
 
+	switch distro {
+	case crcos.RHEL, crcos.CentOS, crcos.Fedora:
+		checks = append(checks, redhatPreflightChecks[:]...)
+	default:
+		logging.Warnf("distribution-specific preflight checks are not implemented for %s", distro)
```

Comment:
This means we'll stop configuring NetworkManager on non EL/Fedora, this will cause regressions on distros which were fine with the NetworkManager configuration we are doing. In this `default` case, I'd fallback to appending `redhatPreflightChecks`  with a warning.

Reply:
I added the same line for the default case.